cuda/zhangli2.cu

```diff
@@ -4,7 +4,8 @@
 #include "stencil.h"
 #include <stdint.h>
 
-#define PREFACTOR ((MUB) / (2 * QE * GAMMA0))
+// #define PREFACTOR ((MUB) / (2 * QE * GAMMA0))
+#define PREFACTOR ((HBAR) / (2 * QE))
```

Comment:
Is this new calculation of `PREFACTOR` correct?

The only place where `PREFACTOR` is used is a bit further in this file, where it is now divided by `g`. As far as I understand it, `g` should be 1 by default, so should `PREFACTOR` then not stay the same?
The previous factor `MUB/GAMMA0 == 5.27082077425405e-35` but the new factor `HBAR == 1.05457173e-34`.

Reply:
Hi Jonathan,

I think you're right... The only place where this `PREFACTOR` is used is on line ~41, which is changed
  (-) `float b = invMs * PREFACTOR / (1.0f + xi*xi);`
  (+) `float b = invMs * PREFACTOR / (g*(1.0f + xi*xi));` 

So, the new value of `PREFACTOR` should be the old one times the default value of `g` (which is 1.),  and not twice its value.

I think the problem came about because initially (and in the code I use) I defined g as the physical Landé g-factor (=2.00 by default) and removed the parameter `GammaLL`. In this case, the new `PREFACTOR` should indeed be twice the old value. This definition makes much more sense, as `g` would be the physical well-known g-factor, but breaks the backwards compatibility in scripts that use the parameter `GammaLL`. I think I might have mixed the files when uploading this change, and I'm surprised it wasn't caught in the validation tests...

~~ João